nexus/api.go

```diff
@@ -145,6 +149,25 @@ func addCallbackHeaderToHTTPHeader(nexusHeader Header, httpHeader http.Header) h
 	return httpHeader
 }
 
+func addLinksToHTTPHeader(links []Link, httpHeader http.Header) error {
+	for _, link := range links {
+		httpHeader.Add(headerLinks, encodeLink(link))
+	}
+	return nil
+}
+
+func getLinksFromHeader(httpHeader http.Header) ([]Link, error) {
+	var links []Link
+	for _, encodedLink := range httpHeader.Values(headerLinks) {
+		link, err := decodeLink(encodedLink)
```

Comment:
I would double check that Go automatically splits up headers that have multiple values (e.g. you won't see commas here).

Reply:
From what I can tell, Go only accepts multi-value header by specifying multiple times the same header name.

Reply:
I checked, Go will split up the header values.